### PR TITLE
New website (docs) with codelabs

### PR DESCRIPTION
**Pull request by nweldev**
Using VuePress, inspired by @open-wc

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
         tar -cvzf www.tar.gz dist
 
     - name: Publish Github Release
-      uses: "marvinpinto/action-automatic-releases@latest"
+      uses: "marvinpinto/action-automatic-releases@v0.3.0"
```
Comment (nweldev) [inline .github/workflows/build-and-deploy.yml]: bad rebase
```suggestion
      uses: "marvinpinto/action-automatic-releases@latest"
```

File: docs/.vuepress/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "The Web on FIRE Show",
+  "short_name": "WoF Show",
```
Comment (noelmace) [inline docs/.vuepress/public/manifest.json]:
```suggestion
  "name": "Web on FIRE",
  "short_name": "WoF",
```

File: docs/.vuepress/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "The Web on FIRE Show",
```
Comment (nweldev) [inline docs/.vuepress/public/manifest.json]:
```suggestion
  "name": "Web on FIRE",
```

File: docs/.vuepress/public/manifest.json (hunk 2 above)
Comment (nweldev) [inline docs/.vuepress/public/manifest.json]:
```suggestion
  "short_name": "WoF",
```

**Pull request merged**